winml/lib/Api/LearningModelBinding.h

```diff
@@ -61,15 +67,18 @@ struct LearningModelBinding : LearningModelBindingT<LearningModelBinding, ILearn
   bool IsOfTensorType(const Ort::Value& ort_value, TensorKind kind);
   bool IsOfMapType(const Ort::Value& ort_value, TensorKind key_kind, TensorKind value_kind);
   bool IsOfVectorMapType(const Ort::Value& ort_value, TensorKind key_kind, TensorKind value_kind);
-
+  HRESULT BindInput(const std::string& name, Ort::Value& ml_value);
 
  private:
   const Windows::AI::MachineLearning::LearningModelSession m_session;
 
   std::unordered_map<std::string, ProviderInfo> m_providers;
 
-  com_ptr<_winmla::IIOBinding> m_lotusBinding;
   com_ptr<_winmla::IWinMLAdapter> adapter_;
+  std::vector<std::string> feed_names_;
```

Comment:
>feed_ [](start = 27, length = 5)

we've been calling these "inputs" not "feeds". can we keep calling them inputs or is there a reason to change?

Reply:
Yep! These can be called inputs. I just preserved the naming from ORT. Let me change it back